packages/core/useVirtualList/index.ts

```diff
@@ -119,21 +119,24 @@ function useVirtualListResourses<T>(list: MaybeRef<T[]>): UseVirtualListResource
   return { state, source, currentList, size, containerRef }
 }
 
-function createGetViewCapacity<T>(state: UseVirtualListResources<T>['state'], source: UseVirtualListResources<T>['source'], itemSize: UseVirtualListItemSize) {
-  return (containerHeight: number) => {
+function createGetViewCapacity<T>(
+  state: UseVirtualListResources<T>['state'],
+  source: UseVirtualListResources<T>['source'],
+  itemSize: UseVirtualListItemSize,
+) {
+  return (containerSize: number) => {
     if (typeof itemSize === 'number')
-      return Math.ceil(containerHeight / itemSize)
+      return Math.ceil(containerSize / itemSize)
 
     const { start = 0 } = state.value
     let sum = 0
     let capacity = 0
-    for (let i = start; i < source.value.length; i++) {
-      const height = itemSize(i)
-      sum += height
-      if (sum >= containerHeight) {
-        capacity = i
+    for (let i = start; i < source.value.length + 1; i++) {
+      const size = itemSize(i)
+      sum += size
+      capacity = i
+      if (sum > containerSize)
```

Comment:
Moving capacity out of the if is one of the fixes. Without this, if the original list size is smaller than what could fit in the view, we end up with a capacity of 1 + overscan

Reply:
Shouldn't the same thing be done with 'offset = i' in function 'createGetOffset'?